docs/reference/how-to/size-your-shards.asciidoc

```diff
@@ -68,7 +68,17 @@ This decreases the number of segments, which means less metadata is kept in
 heap memory.
 
 Every mapped field also carries some overhead in terms of memory usage and disk
-space. By default {es} will automatically create a mapping for every field in
+space. While the exact amount resource usage of each additional field depends
+on the type of field, an additional 1024B in memory use  for each additional
+mapped field on data nodes is a good approximation in most cases.
+As a result, the more mapped fields you have per index, the fewer
+indices and shards each individual data node will be able to hold,
+all other things being equal. In addition to the per field overhead you should
+at least have an additional 512MB of heap available per data node.
+For example, holding 1000 indices that each contain 4000 fields will require
+1000 × 4000 × 1024B = 4GB for the fields and another 512MB for a total of
+at least 4.5GB on the data node.
```

Comment:
```suggestion
at least 4.5GB on the data node. As a reminder, 4.5GB heap here refers to holding the idling data 
in heap on the data node only. Heap usage on data nodes is also heavily dependent on use cases  
as well as how often these shards are indexed, updated or queried, and if there is heavy 
aggregation performed on the data nodes. The max number of shards a node can hold should be 
 adjusted by taking your usage pattern into consideration.
```

How can we emphasise this is for idling shards only? Maybe add a sentence like this?